plugins/ooni/main.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "OONI",
+    "name": "ooni",
```

Comment:
Ah, so the problem is that in plugins the folder is ooni rather than OONI named?